src/documentation/tutorials/index.md

```diff
@@ -15,6 +15,7 @@ page_id: tutorials
 * {% id_link getting-started-buzzer-deck %}
 * {% id_link getting-started-z-ranger-deck %}
 * {% id_link getting-started-flow-deck %}
+* {% id_link solder-deck-pins %}
```

Comment:
Doubting if we should put this in these beginner tutorials at all... it voids warranty and we don't want it to appear to be the 'standard'.

We should remove it, as it is already referred to in https://www.bitcraze.io/documentation/system/platform/cf2-expansiondecks/#additional-information

Reply:
alright fair enough, but we should put at least a big fat warning on top about that we are not able to take back these products in case it is broken.